zone/worldserver.cpp

```diff
@@ -1548,6 +1548,16 @@ void WorldServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p)
 		r->SendRaidMOTD();
 		break;
 	}
+	case ServerOP_RaidNote: {
+		auto snote = (ServerRaidNote_Struct*)pack->pBuffer;
+		if (snote->rid > 0) {
+			Raid* r = entity_list.GetRaidByID(snote->rid);
+			if (!r)
```

Comment:
Needs curlies.

Reply:
Rewrote to be the affirmative and placed curlies.